### PR TITLE
NUnitProject should parse .nunit project files containing Xml Declarations

### PR DESCRIPTION
**Pull request by teyc**
I was handcrafting an .nunit project and had included the `<?xml version="1.0"?>` header in the file, and nunit3-console was unable to parse it properly, as `firstChild` refers to the xml declaration node. A one line fix to use the `documentElement` instead was sufficient to fix the bug.

I've included a test case illustrating the issue.

### CODE REVIEW

**Comment by rprouse** [comment]
Looks good to me. I am assigning it to a milestone so that it will get picked up in the release notes after it is merged.

File: src/NUnitEngine/Addins/addin-tests/NUnitProjectLoaderTests.cs [hunk 1]
```diff
@@ -86,10 +86,12 @@ public void LoadEmptyConfigs()
             }
         }
 
+        [TestCase("NUnitProject.nunit")]
+        [TestCase("NUnitProject_XmlDecl.nunit")]
         [Test]
```
Comment (CharliePoole) [inline src/NUnitEngine/Addins/addin-tests/NUnitProjectLoaderTests.cs]: TestAttribute is now redundant. As a matter of style, we don't normally use it wiith test cases.

File: src/NUnitEngine/Addins/nunit-project-loader/NUnitProject.cs [hunk 2]
```diff
@@ -156,7 +156,7 @@ public string GetSetting(string name, string defaultValue)
         /// </summary>
         internal XmlNode RootNode
         {
-            get { return xmlDoc.FirstChild; }
+            get { return xmlDoc.DocumentElement ; }
```
Comment (CharliePoole) [inline src/NUnitEngine/Addins/nunit-project-loader/NUnitProject.cs]: Good pickup. There are other places where we use FirstChild and should probably use DocumentElement.

**Comment by CharliePoole** [comment]
Nice catch! And good on you for adding a test for it. :-)

@rprouse I think it's mergeable either as is or with fixes noted.

**Comment by rprouse** [comment]
@teyc thanks for merging latest from master. I was just about to do that :smile: 

If you get a chance, can you remove the extra `[Test]` that @CharliePoole pointed out? If not, I will fix it.

**Pull request merged**